src/features/intl-displaynames.md

```diff
@@ -0,0 +1,112 @@
+---
+title: '`Intl.DisplayNames`'
+author: 'Shu-yu Guo ([@_shu](https://twitter.com/_shu)) and Frank Tang'
+avatars:
+  - 'shu-yu-guo'
+  - 'frank-tang'
+date: 2020-02-13
+tags:
+  - Intl
+  - Node.js 14
+description: 'The Intl.DisplayNames API enables localized names of languages, regions, scripts, and currencies.'
+---
+Web applications that reach a global audience need to show the display names of languages, regions, scripts, and currencies in many different languages. The translations of those names require data, which is available in the [Unicode CLDR](http://cldr.unicode.org/translation/). Packaging the data as part of the application incurs a cost on developer time. Users are likely to prefer consistent translations of language and region names, and keeping that data up to date with the world's geopolitical happenings requires constant maintenance.
+
+Luckily, most JavaScript runtimes already ship and keep up-to-date that very same translation data. The new `Intl.DisplayNames` API gives JavaScript developers direct access to those translations, allowing applications to more easily display localized names.
+
+## Usage examples
+
+The following example shows how to create an `Intl.DisplayNames` object to get region names in English using [ISO-3166 2-letter country codes](https://www.iso.org/iso-3166-country-codes.html).
+
+```js
+const regionNames = new Intl.DisplayNames(['en'], { type: 'region' });
+regionNames.of('US');
+// → 'United States'
+regionNames.of('BA');
+// → 'Bosnia & Herzegovina'
+regionNames.of('MM');
+// → 'Myanmar (Burma)'
+```
+
+The following example gets language names in Traditional Chinese using [Unicode's language identifier grammar](http://unicode.org/reports/tr35/#Unicode_language_identifier).
+
+```js
+const languageNames = new Intl.DisplayNames(['zh-Hant'], { type: 'language' });
+languageNames.of('fr');
+// → '法文'
+languageNames.of('zh');
+// → '中文'
+languageNames.of('de');
+// → '德文'
+```
+
+The following example gets currency names in Simplified Chinese using [ISO-4217 3-letter currency codes](https://www.iso.org/iso-4217-currency-codes.html).
+
+```js
+const currencyNames = new Intl.DisplayNames(['zh-Hans'], {type: 'currency'});
```

Comment:
Are these for singular currency names of plural ones? Is there a way to customise this via options? (seems like an obvious question reader would ask)

Reply:
Well, depends on the language, right? For those that have singular/plural split, it's singular. For languages that have cases, it's nominative IIRC. This API doesn't return plurals for languages that have them.

Reply:
What's the action item you're asking for here? Clarification that it's only singular names in the text?

Reply:
Oh, and also, to get at the plural long-form currency names, [Intl.NumberFormat](https://v8.dev/features/intl-numberformat) can be used.

Reply:
> Clarification that it's only singular names in the text?

Yeah, if that's the case. Probably worth including a note about `NumberFormat` (and a link to that other explainer) as well.

Reply:
...which you already did :)